src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/CustomUriConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public class CustomUriConverter : JsonConverter<Uri>
+    {
+        public override Uri Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+                return new Uri(reader.GetString(), UriKind.RelativeOrAbsolute);
+            if(reader.TokenType == JsonTokenType.StartObject)
+            {
+                if (reader.Read())

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)